### PR TITLE
Fixes #34 - `jubjub` dep unset `default-features`

### LINKED ISSUE #34: Wrong dependency flag for jubjub
`dusk-pki` is importing `jubjub` without `default-features=false`. This cause the dependencies of `pki` to break for `no-std` environments

### PR DESCRIPTION
**Pull request by vlopes11**
Fixes #34

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.5.2] - 2021-02-09
+
+### Changed
+
+- Fix `jubjub` default-features flag [#34](https://github.com/dusk-network/dusk-pki/issues/34)
```
Comment (CPerezz) [inline CHANGELOG.md]: I would say something like: Disable default features for `dusk-jubjub` dep by default.

**Review by CPerezz — APPROVED** [review]
I would maybe rephrase the Changelog. 

Aside from that, all good.

File: CHANGELOG.md (hunk 1 above)
Comment (vlopes11) [reply CHANGELOG.md]: Agreed, fixed
